libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -163,7 +171,7 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @Values: 0:AUTO,1:PX4V1,2:Pixhawk,3:Pixhawk2,4:Pixracer,5:PixhawkMini,6:Pixhawk2Slim,7:VRBrain 5.1,8:VRBrain 5.2,9:VR Micro Brain 5.1,10:VR Micro Brain 5.2,11:VRBrain Core 1.0,12:VRBrain 5.4,13:Intel Aero FC,20:AUAV2.1
     // @RebootRequired: True
     // @User: Advanced
-    AP_GROUPINFO("TYPE", 9, AP_BoardConfig, px4.board_type, BOARD_TYPE_DEFAULT),
+    AP_GROUPINFO("TYPE", 10, AP_BoardConfig, px4.board_type, BOARD_TYPE_DEFAULT),
```

Comment:
same with these - basic rule is never change an index number of an existing parameter